### ISSUE TITLE
[BUG] CRITICAL: Links to datasets broken 

### ISSUE THREAD

**Issue by kevin931** [kevin931/PyCytoData#3]
## What issues are you experiencing?
The method to download datasets no longer works properly because the backend of HDCytoData is broken. Users will no longer be able to download one of the three cohorts of CyTOF data from PyCytoData.

## To Reproduce
The following snippet from documentation will not work:

```python
>>> from PyCytoData import DataLoader
>>> exprs = DataLoader.load_dataset(dataset = "levine13")
Would you like to download levine13? [y/n]y

Download in progress...
This may take quite a while, go grab a coffee or cytomulate it!

Traceback (most recent call last):
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 1354, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 1256, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 1302, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 1251, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 1011, in _send_output
    self.send(msg)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 951, in send
    self.connect()
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/http/client.py", line 922, in connect
    self.sock = self._create_connection(
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
TimeoutError: [Errno 110] Connection timed out

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/PyCytoData/PyCytoData/data.py", line 864, in load_dataset
    cls._download_data(dataset = dataset, force_download = force_download)
  File "/PyCytoData/PyCytoData/data.py", line 935, in _download_data
    contents = urlopen(url)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 525, in open
    response = self._open(req, data)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 542, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 502, in _call_chain
    result = func(*args)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 1383, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/home/kevin/anaconda3/envs/cytof/lib/python3.8/urllib/request.py", line 1357, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno 110] Connection timed out>
```

Indeed, the link **does not** work on a browser either. 

## Your environment:
 - OS: Windows 11
 - Python Distribution: Anaconda
 - PyCytoData distribution: Anaconda

**Comment by kevin931** [comment]
This issue is fixable, I believe. There are alternative hosting/availabilities of the public datasets, but the format may need to be closely inspected and potentially reworked. Exact backwards compatibilities can also be a bit tricky.

I will PR at some point when I get around to attempt this.

**Comment by kevin931** [comment]
I've finally gotten around to fix this amid delays (See PR #14).  It was surprisingly straightforward in terms of implementation. Here is a summary of what was used to fix it and what it entails:
- We no longer rely on an external website from ``HDCytoData``.
- Now, the datasets are hosted as GitHub releases, which should be much more stable.
- There should be no breaking change. 

The new changes will be released soon (v0.1.2). In the meantime and after the release, hope all things go well! If issues persist, please leave comments here and I will address them accordingly.
